src/formElementPlugins/password/src/ApplicationView.tsx

```diff
@@ -91,25 +103,27 @@ const ApplicationView: React.FC<ApplicationViewProps> = ({
             : null
         }
       />
-      <Form.Input
-        name="passwordConfirm"
-        fluid
-        placeholder={confirmPlaceholder ? confirmPlaceholder : 'Confirm password'}
-        onChange={handleChange}
-        onBlur={handleLoseFocus}
-        onFocus={setIsActive}
-        value={passwordConfirm ? passwordConfirm : ''}
-        disabled={!isEditable}
-        type={masked ? 'password' : undefined}
-        error={
-          passwordConfirm !== password && passwordConfirm !== ''
-            ? {
-                content: 'Passwords do not match',
-                pointing: 'above',
-              }
-            : null
-        }
-      />
+      {requireConfirmation && (
+        <Form.Input
+          name="passwordConfirm"
+          fluid
+          placeholder={confirmPlaceholder ? confirmPlaceholder : 'Confirm password'}
+          onChange={handleChange}
+          onBlur={handleLoseFocus}
+          onFocus={setIsActive}
+          value={passwordConfirm ? passwordConfirm : ''}
+          disabled={!isEditable}
+          type={masked ? 'password' : undefined}
+          error={
+            passwordConfirm !== password && passwordConfirm !== ''
+              ? {
+                  content: 'Passwords do not match',
```

Comment:
Hmmm I wonder how to localise things that are internal to plugins...

Reply:
We have constants.ts files for each plugin, although most aren't using it (Checkbox has one). I can move this one into a localisation file for this plugin too.

Reply:
Done 9c80005b